contracts/license/src/state.rs

```diff
@@ -79,11 +78,14 @@ impl LicenseContractState {
 
     /// Returns licenses for a given range of block-heights.
     /// Method intended to be called by the user.
-    pub fn get_licenses(&mut self, block_heights: Range<u64>) -> Vec<Vec<u8>> {
+    pub fn get_licenses(
+        &mut self,
+        block_heights: Range<u64>,
+    ) -> Vec<(u64, Vec<u8>)> {
```

Comment:
Not strictly related to this PR, but you should consider to use the feeder query when returning collection of elements.
Just to avoid sending all data together